Makefile

```diff
@@ -100,6 +100,12 @@ run-openapi-demo: generate-test-keys generate-openapi-demo-specs
 	@DEMO_COMPOSE_PATH=test/bdd/fixtures/demo/openapi SIDETREE_COMPOSE_PATH=test/bdd/fixtures/sidetree-mock AGENT_REST_COMPOSE_PATH=test/bdd/fixtures/agent-rest  \
         scripts/run-openapi-demo.sh
 
+.PHONY: stop-openapi-demo
+stop-openapi-demo: generate-test-keys generate-openapi-demo-specs
```

Comment:
don't need `generate-test-keys` nor `generate-openapi-demo-specs`

Reply:
Whoops, that sloppy.